home-page

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Binabox NFT Collection</title>
+</head>
+<body>
+
+    <h2>Binabox</h2>
+    <p><a href="#">Home</a></p>
+    <p><a href="#">About</a></p>
+    <p><a href="#">Roadmap</a></p>
+    <p><a href="#">Collection</a></p>
+    <p><a href="#">FAQs</a></p>
+    <p><a href="#">Pages</a></p>
+
+    <br>
+
+    <p><a href="#">Discord</a> | <a href="#">Connect</a></p>
```

Comment:
This vertical slash will be added at the expense of css, it is a decorative element, it does not apply to markup